src/npctalk.cpp

```diff
@@ -1342,7 +1303,8 @@ void dialogue::gen_responses( const talk_topic &the_topic )
             const int next_level_exercise = skill_level_obj.exercise();
 
             //~Skill name: current level (exercise) -> next level (exercise) (cost in dollars)
-            std::string text = string_format( cost > 0 ? _( "%s: %d (%d%%) -> %d (%d%%) (cost $%d)" ) :
+            std::string text = string_format( cost > 0 ?
+                                              _( "%s: %d (%d%%) -> %d (%d%%) (cost $%d)" ) :
```

Comment:
This broke the link to the above translator comment.
```suggestion
            std::string text = string_format( cost > 0 ? _( "%s: %d (%d%%) -> %d (%d%%) (cost $%d)" ) :
```